pkg/git/repository.go

```diff
@@ -36,14 +37,15 @@ type (
 	}
 
 	CloneOptions struct {
-		// URL clone url
-		Repo     string
-		Auth     Auth
-		FS       fs.FS
-		Progress io.Writer
-		url      string
-		revision string
-		path     string
+		AutoCreate bool
+		Provider   string
```

Comment:
Do we want to support creation of public repo?

Reply:
I didn't want to add yet another flag to the bootstrap command. I imagine the only times people would want it in a public repo would be for demo purposes. In such cases they can either create the repo themselves, or change its visibility afterwards.
The opinionated autopilot can just create it as private, imo.

Reply:
Makes sense I guess.